src/style.css

```diff
@@ -0,0 +1,37 @@
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  height: 16px;
+  width: 16px;
+  margin-right: 4px;
+  background-image: url("./images/star.svg");
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.stars--5 .stars__star {
+  background-image: url("./images/star-active.svg");
+
+}
+
+.stars--1 .stars__star:first-child {
+  background-image: url("./images/star-active.svg");
+}
+
+.stars--2 .stars__star:nth-child(-n+2) {
+  background-image: url("./images/star-active.svg");
+}
+
+.stars--3 .stars__star:nth-child(-n+3) {
+  background-image: url("./images/star-active.svg");
+}
+
+.stars--4 .stars__star:not(:last-child) {
+  background-image: url("./images/star-active.svg");
+}
```

Comment:
Second point in the [checklist](https://github.com/mate-academy/layout_stars/blob/master/checklist.md)